pkgs/top-level/all-packages.nix

```diff
@@ -24072,6 +24072,8 @@ with pkgs;
 
   artha = callPackage ../applications/misc/artha { };
 
+  athens-beta-bin = callPackage ../applications/misc/athens/beta.nix { };
```

Comment:
Is there an athens or athens-beta package? If not we would just short it to athens. Also if a directory only contains one nix file it is usually called default.nix which can also be ommited.

Reply:
No, but the sources are available.

I guess that it could be named athens or athens-beta and if someone would manage to build it from source we would just drop the appimage code and use the same name.

For the beta part, 1.0.0 was released 15 days ago, but I'm guessing a good part of the user base would want to use the beta, which has "collaboration support" with a server.

What do you think?

Reply:
I don't have an opinion about what version is packaged. I guess it is fine.